chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/presenter/CertifiedProductCsvPresenter.java

```diff
@@ -109,7 +108,6 @@ public void setFf4j(FF4j ff4j) {
         result.add("Version");
         result.add("Version Database ID");
         if (ff4j.check(FeatureList.RWT_ENABLED)) {
-            result.add("Real World Testing Eligibility Year");
```

Comment:
If this file changed, then the CSV definition file for this type of data probably needs to change too